src/libs/tmdk/package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "test": "./node_modules/.bin/jest --silent",
     "integ-test": "./node_modules/.bin/jest --silent -c jest.config.integration.js",
+    "e2e-test": "./node_modules/.bin/jest -c jest.config.e2e.js",
```

Comment:
instead of creating a new `e2e-test` can we just udpdate `integ-test`?

Reply:
We discussed with AppKit engineers that integ tests will be mocked through jest and verify api calls rather than validate state. I feel this test is better labeled as an e2e test. I agree having 3 types of tests is confusing, but I think the classifications are:
Unit tests: verify simple edge cases and method returns
Integ tests: verify api calls and expected behavior
E2E test: simulate user interaction and verify interaction with AWS

Reply:
ok, I think we can delete `integ-test` for now if it means something specific in AppKit and call these "functional-tests" instead

Reply:
Got it, will update. So for now our current testing naming is:
Unit tests: verify api calls and edge cases
Functional tests: verify e2e user experience and AWS interaction